static/styles/subscriptions.scss

```diff
@@ -556,18 +556,35 @@ form#add_new_subscription {
     justify-content: center;
     margin-bottom: 0;
     height: auto;
+    border-bottom: 1px solid hsl(0, 0%, 87%);
+}
+
+.streams-list {
```

Comment:
I tried to test the 1st commit with these changes, and couldn't see any issues. Can you explain what the problem was? A GIF or screenshot of the problem, and steps to reproduce it would bre great.

Reply:
## GIFS

### BEFORE (on commit 896fb6cda9e1b7fd90699b1f3844fa6896faab17):
![simplebar-flaw](https://user-images.githubusercontent.com/43912285/77229292-1a226080-6bb3-11ea-8cb4-1e3d360a9ae2.gif)

_Notice how the stream titled Verona is not entirely visible._

### AFTER (on commit a9e09d9156bb933d1f6bc327b6138230a7391f23)
![simplabar-fixed](https://user-images.githubusercontent.com/43912285/77229500-66ba6b80-6bb4-11ea-92db-d88de17feca5.gif)

## Steps to reproduce
1. Checkout commit 896fb6cda9e1b7fd90699b1f3844fa6896faab17
2. Open the streams settings modal after logging in as any user
3. Adjust your viewport width so that the filter streams search box appears in a new line. Try scrolling the list of streams below

## The issue
The height of the `.streams-list` element was being calculated as `calc(100% - 45px)` which was only accounting for the height of the original search box. When the search box shifts to a new line, we need to account for its height as well. Without this change, a part of the element isn't visible in the modal's body, and hence it appears cut off.